README.md

```diff
@@ -1 +1,3 @@
-# graasp-template-backend
+# graasp-plugin-h5p
+
+This Graasp plugin provides back-end support for [H5P](https://h5p.org) packages through a Fastify plugin
```

Comment:
Should we add more links here like the h5p website, standalone packages, everything you needed to build this plugin?

Reply:
Would like to see also a list of all the problems during development (I think cookies, static files, etc)

Reply:
And we don't have a GET call because it's a public bucket and it's handled by the reader, right?